hyperion-core/consumer-proguard-rules.pro

```diff
@@ -1 +1,5 @@
--keep class * implements com.willowtreeapps.hyperion.plugin.v1.Plugin
\ No newline at end of file
+-keep class * implements com.willowtreeapps.hyperion.plugin.v1.Plugin { *; }
+-keep class com.willowtreeapps.hyperion.plugin.v1.Plugin { *; }
+-keep class com.willowtreeapps.hyperion.attr.collectors.TypedAttributeCollector { *; }
```

Comment:
I would prefer if the TypedAttributeCollector keep was part of the `hyperion-attr` plugin's consumer proguard file.